dangerfile.js

```diff
@@ -44,8 +34,41 @@ if ( results.warnings.length > 0 || results.fails.length > 0 ) {
 	markdown(
 `**Thank you for the great PR description!**
 
-When this PR is ready for review, please apply the \`[Status] Needs Review label\`. If you are an a11n, please have someone from your team review the code if possible. The Jetpack team will also review this PR and merge it to be included in the next Jetpack release.
+When this PR is ready for review, please apply the \`[Status] Needs Review label\`. If you are an a11n, please have someone from your team review the code if possible. The Jetpack team will also review this PR and merge it to be included in the next Jetpack release.` );
+	setReleaseDates();
+}
+
+// Adds release and code freeze dates according to x.x milestone due date
+function setReleaseDates() {
+	schedule( async () => {
+		let jetpackReleaseDate;
+		let codeFreezeDate;
+		const milestones = await github.api.issues.getMilestones( { owner: 'Automattic', repo: 'jetpack' } );
+
+		// Find a milestone which name is a version number
+		// and it's due dates is earliest in a future
+		const nextMilestone = milestones.data
+		.filter( m => ! isNaN( m.title ) )
```

Comment:
Will this work for version `6.8.1`?

Reply:
nice catch! Attempted to fix that in ceb724d. Although, I don't think x.x.x versions should be mentioned in Danger bot messages